api_spec.yaml

```diff
@@ -615,7 +619,22 @@ paths:
                     properties:
                       status:
                         type: string
-                        enum: ['SCOPE_DOES_NOT_EXIST']
+                        enum: ['UNKNOWN_SCOPE_ERROR']
+                      scope:
+                        $ref: '#/components/schemas/oauth2Scope'
+                  - type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ['UNKNOWN_CLIENT_ID_ERROR']
+                  - type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: [SCOPE_ALREADY_EXISTS_ERROR]
```

Comment:
Therefore, the `newScopes` output of status OK is not needed either.

Reply:
as far i understand, this api isn't used that frequently.. So it would better to allow multiple scopes addition and removal from flexibility point of view.

Reply:
changes made